cves/CVE-2016-8747.yml

```diff
@@ -185,8 +188,8 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: Apache while refactoring this section of code to make the system fast at handleing requests, did not put thought into the security implications of such changes which let to the system to leak other requests information
```

Comment:
spelling error in "handling", and consider rewording this note, it comes across as difficult to read